### PR TITLE
[13.0][IMP] ebill_paynet: round taxes

### PR DESCRIPTION
**Pull request by i-vyshnevska**
IMO rounding shouldn't be done on jinja template data should be corrected on invoice level but as the other part implemented in this way we follow common approach

### CODE REVIEW

**Review by sebalix — COMMENTED** [review]
pre-commit is failing

File: ebill_paynet/messages/invoice-2003A.xml [hunk 1]
```diff
@@ -210,7 +210,7 @@
       {%- else %}
       <Rate Category="E">0</Rate>
       {%- endif %}
-      <Amount Currency="{{ invoice.currency_id.name }}">{{ taxgroup[1] }}</Amount>
+      <Amount Currency="{{ invoice.currency_id.name }}">{{ taxgroup[1]|round(2) }}</Amount>
```
Comment (simahawk) [inline ebill_paynet/messages/invoice-2003A.xml]: can't we pass a function to handle this instead?

**Review by sebalix — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@i-vyshnevska can you write a better commit msg? eg `ebille_paynet: round taxes` or similar? FTR https://www.odoo.com/documentation/15.0/developer/misc/other/guidelines.html#commit-message-structure

**Review by TDu — APPROVED** [review]

**Pull request merged**